packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -352,39 +356,6 @@ export function isTaggedTelemetryPropertyValue(
 	return typeof (x as Partial<Tagged<unknown>>)?.tag === "string";
 }
 
-/**
```

Comment:
Replaced these removed functions with `convertToBasePropertyType()` [here](https://github.com/alexvy86/FluidFramework/blob/99dac5b99f483fc62f6321a04d70dddee638dac3/packages/utils/telemetry-utils/src/logger.ts#L870-L912), which has a more up-to-date version of the relevant logic.

Reply:
Oh... now that I'm thinking more carefully about this PR, I think the extra care here is intentional. But maybe we can reconsider. I'll explain.

The other utility you linked to is used only for logging events.  This function is used for the result of `getTelemetryProperties`, which includes ALL properties off the error object itself (unless specifically omitted). So the idea was to avoid logging huge/complex objects and only log the simple kinds of stuff we expected.

However, we're moving towards logging more and more complex objects - there's been a recent request to log an array of records... at that point we might just say you can log anything JSONable. In other words - There is no harm if a non-compliant property gets stringified, and if it's not desirable there's a quick mitigation (add to `omitPropsFromLogging`).

The only reason to pause would be a privacy concern - the word "REDACTED" sounds importantm right? If we were passing arbitrary Error objects here then yeah we'd be concerned (e.g. fetch could stick the whole request object on the error and then boom we're logging auth headers), but this is only used for LoggingError, which we always create ourselves and have an understanding that all its props will be logged.

Reply:
On that last note (that this codepath only applies to LoggingErrors), since you're updating the free function `getValidTelemetryProps` to remove this protection, can you move it to be a private function on `LoggingError`? Or maybe just inline it, dunno up to you.

Reply:
I'd even say we can just get rid of `ILoggingError`.  It is not serving a purpose being internal, and 2 years ago when I suggested to a 1P partner that they leverage it to log all our error props they didn't want to (didn't trust us not to leak private info I think - silly because we're logging it anyway!).  Anywhere using `isILoggingError` can be replaced with `LoggingError.typeCheck`.

Up to you if you do that in this PR or not, you have license to do so though IMO.